src/ripple/basics/impl/StringUtilities.cpp

```diff
@@ -31,14 +32,11 @@
 
 namespace ripple {
 
-// NIKB NOTE: This function is only used by strUnHex (std::string const& strSrc)
-// which results in a pointless copy from std::string into std::vector. Should
-// we just scrap this function altogether?
-int strUnHex (std::string& strDst, std::string const& strSrc)
+std::pair<Blob, bool> strUnHex (std::string const& strSrc)
```

Comment:
Except that the boost routines return an iterator instead of a container, making them clumsy to use. And they throw exceptions. We would have to wrap them in our own function. Say, a function called `strUnHex`...